test/utils.sh

```diff
@@ -157,18 +157,24 @@ parse_test_output() {
   fi
 }
 
-# the function will be used by the tekton tasks of build-definitions
+# The function will be used by the tekton tasks of build-definitions
+# It need a tekton result name as parameter
```

Comment:
```suggestion
# It need a tekton result path as parameter
```

Reply:
Yes, it can accept file path. I must have been dizzy.

Reply:
Fixed.